### PR TITLE
Simple-singolo

### PR DESCRIPTION
**Pull request by hauteballe**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+                    <a class='side-menu-item' href="#home">Home</a>
+                    <a class='side-menu-item' href="#services">Services</a>
+                    <a class='side-menu-item' href="#portfolio">Portfolio</a>
+                    <a class='side-menu-item' href="#contact">About</a>
+                    <a class='side-menu-item' href="#contact">Contact</a>
+               </div>
+            </div>
+        </div>
+        <header>
+            <div class="wrapper">
```
Comment (tibmt) [inline index.html:37]: You don't need this div. Wrapper styles move to header.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+                <div class="row">
+                    <div class="burger-button">
+                        <img src="assets/img/Burger-menu-icon.png" alt="burger-button">
+                    </div>
+                    <div class="logo" name="home">
+                        <h1>
```
Comment (tibmt) [inline index.html:43]: You should have only one `h1` tag on the page so change it to `h2`

File: style.css [hunk 3]
```diff
@@ -0,0 +1,493 @@
...
+  scroll-behavior: smooth;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0px;
+  font-family: "Lato", Arial, Helvetica, sans-serif;
```
Comment (tibmt) [inline style.css:11]: You can unite `html`, `*` and `body` styles

File: style.css [hunk 4]
```diff
@@ -0,0 +1,493 @@
...
+}
+
+.logo h1 {
+  font-size: 25px;
+  font-weight: bold;
+  text-transform: uppercase;
+  letter-spacing: -0.03em;
+}
+
+.logo h1 span {
```
Comment (tibmt) [inline style.css:47]: I propose add some class name for this

File: scripts.js [hunk 5]
```diff
@@ -0,0 +1,164 @@
+/* side-menu */
+
+function burgerClickHandler(ev) {
```
Comment (tibmt) [inline scripts.js:3]: You dan't use `ev` parametr in function so you can remove it.

File: scripts.js [hunk 6]
```diff
@@ -0,0 +1,164 @@
...
+    let popup = document.getElementsByClassName('menu-popup')[0];
+    popup.style.display = 'block';
+}
+
+function closeSideMenu(ev) {
```
Comment (tibmt) [inline scripts.js:8]: The same as above

File: scripts.js [hunk 7]
```diff
@@ -0,0 +1,164 @@
...
+    sliderControlLeft.addEventListener('click', handleControleLeft);
+    sliderControlRight.addEventListener('click', handleControleRight);
+}
+  
+let slider = Slider('.slider') 
+  
+
+/* portfolio-tabs */
+
+let images = [
```
Comment (tibmt) [inline scripts.js:123]: Move this constant to `const.js` file

File: scripts.js [hunk 8]
```diff
@@ -0,0 +1,164 @@
...
+    "assets/img/portfolio-9.png",
+    "assets/img/portfolio-10.png",
+    "assets/img/portfolio-11.png",
+    "assets/img/portfolio-12.png",
+    "assets/img/portfolio-10.png",
+    "assets/img/portfolio-11.png",
+    "assets/img/portfolio-12.png"
+]
+
+function choose(choices) {
```
Comment (tibmt) [inline scripts.js:141]: Choose what?

File: index.html (hunk 1 above)
Comment (hauteballe) [reply index.html:37]: I use this div for controlling max-width, it's a common class for all the sections on the page.

Comment (tibmt) [reply index.html:37]: Good in such case

File: index.html (hunk 2 above)
Comment (hauteballe) [reply index.html:43]: In this case, h1 tag is used once for the desktop version and once for the mobile version. Is it right or should I change it in the second case?

File: style.css (hunk 3 above)
Comment (hauteballe) [reply style.css:11]: Could you explain in more details?

File: style.css (hunk 4 above)
Comment (hauteballe) [reply style.css:47]: Done

File: style.css (hunk 3 above)
Comment (tibmt) [reply style.css:11]: you can do like this
`* {

   all styles here

}`

File: scripts.js (hunk 7 above)
Comment (hauteballe) [reply scripts.js:123]: Done

File: scripts.js (hunk 8 above)
Comment (hauteballe) [reply scripts.js:141]: This function was renamed to chooseRandomItem

File: index.html (hunk 2 above)
Comment (tibmt) [reply index.html:43]: It's better to have one `h1` tag on the page

Comment (hauteballe) [reply index.html:43]: Ok, I'll change it

File: style.css (hunk 3 above)
Comment (hauteballe) [reply style.css:11]: Done

**Pull request merged**